### ISSUE TITLE
Windows 10 issue with browserify | ./bin/es3ify.js

### ISSUE THREAD

**Issue by schobbent** [pouchdb/plugin-seed#10]
I had a error with the pipeline :

browserify index.js | ./bin/es3ify.js | derequire > dist/pouchdb.mypluginname.js
./bin/es3ify.js wasn't a recommanded command. It looks like, it wasn't detected and included as a script in my prompt.
To solve that problem I implemented Gulp which has a es3ify module + a uglify module.
So now, the es3ify pipe and uglify pipe is done in gulp.

The commit : https://github.com/schobbent/plugin-seed/commit/77566c3b8853dfc801a3ab3aff0900194513eb0f

**Comment by schobbent** [comment]
Ok, I had to do some gulp for JSHint too.

https://github.com/schobbent/plugin-seed/commit/5840c5044c5620a0c2701c32636516ea434ddc63

**Comment by nolanlawson** [comment]
Yeah, this plugin seed works for a very particular setup (Browserify, Bash, etc.). It's designed to get you started, but you may have to tweak it to fit your needs.

We may end up removing es3ify and derequire since not everybody's going to want to use those things.
